src/components/Table/hoc/withTableSettings/TableColumnSetup/DndTreeSelect.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import {Grip} from '@gravity-ui/icons';
+import {DragDropContext, Draggable, Droppable} from 'react-beautiful-dnd';
+import type {OnDragEndResponder} from 'react-beautiful-dnd';
+
+import {useUniqId} from '../../../../../hooks';
+import {Icon} from '../../../../Icon';
+import {TreeSelect} from '../../../../TreeSelect/TreeSelect';
+import {TreeSelectItem} from '../../../../TreeSelect/TreeSelectItem';
+import type {TreeSelectItemProps} from '../../../../TreeSelect/TreeSelectItem';
+import type {RenderContainerType, RenderItem, TreeSelectProps} from '../../../../TreeSelect/types';
+import {ListContainerView} from '../../../../useList';
+import {reorderArray} from '../../../../useList/__stories__/utils/reorderArray';
+
+export type DndTreeSelectItem = TreeSelectItemProps & {
+    id: string;
+    isDragDisabled?: boolean;
+};
+
+export type RenderDndContainer = (container: {
+    renderList: () => React.JSX.Element;
+}) => React.ReactElement;
+
+const DEFAULT_RENDER_CONTAINER: RenderDndContainer = ({renderList}) => renderList();
```

Comment:
why you overrider render container method he? it looks redundant.

Reply:
I wrote with an eye to the future. 

This place seems very inconsequential, to be honest.